native/swift/Sources/wordpress-api-combine/Stream.swift

```diff
@@ -0,0 +1,58 @@
+import Combine
+import WordPressAPI
+
+public extension PaginationAwareExecutor {
+    func streamWithEditContext(
+        params: Self.EditContextResponseType.ParamsType
+    ) -> Stream<Self, Self.EditContextResponseType> {
+        Stream(executor: self, params: params, sequence: self.sequenceWithEditContext(params: params) )
+    }
+
+    func streamWithViewContext(
+        params: Self.ViewContextResponseType.ParamsType
+    ) -> Stream<Self, Self.ViewContextResponseType> {
+        Stream(executor: self, params: params, sequence: self.sequenceWithViewContext(params: params) )
+    }
+
+    func streamWithEmbedContext(
+        params: Self.EmbedContextResponseType.ParamsType
+    ) -> Stream<Self, Self.EmbedContextResponseType> {
+        Stream(executor: self, params: params, sequence: self.sequenceWithEmbedContext(params: params) )
+    }
+}
+
+public protocol Fetchable {
+    mutating func fetch() async throws
+}
+
+public struct Stream<Executor: PaginationAwareExecutor, ResponseType: PaginatableResponse>: Fetchable {
+    private let publisher = CurrentValueSubject<[ResponseType.DataType], Error>([])
+    private let executor: Executor
+    private let params: ResponseType.ParamsType
+    private let sequence: PaginationSequence<ResponseType>
+
+    private var accumulatedObjects: [ResponseType.DataType] = []
+
+    public init(executor: Executor, params: ResponseType.ParamsType, sequence: PaginationSequence<ResponseType>) {
+        self.executor = executor
+        self.params = params
+        self.sequence = sequence
+    }
+
+    public func getPublisher() -> AnyPublisher<[ResponseType.DataType], Error> {
+        publisher.eraseToAnyPublisher()
+    }
+
+    public mutating func fetch() async throws {
+        self.publisher.send(self.accumulatedObjects)
+
+        do {
+            for try await objects in sequence {
+                accumulatedObjects.append(contentsOf: objects)
+                publisher.send(accumulatedObjects)
+            }
+        } catch {
+            publisher.send(completion: .failure(error))
+        }
+    }
+}
```

Comment:
The fact that the publisher doesn't emit elements unless `fetch` is called doesn't follow combine's programming model.

I don't think we really need this combine library/target, because it doesn't really relate to WordPressAPI. Its core is more like a generic solution of wrapping `AsyncSequence` in `Publisher`.

Reply:
Yeah, agreed – we'll need some support for this on the Rust side, so I've removed it for now in [1900424](https://github.com/Automattic/wordpress-rs/pull/382/commits/190042435c423abf55e6f19e275f87037bce1f64)